### PR TITLE
Refactor all flickr entities

### PR DESCRIPTION
**Pull request by st0o0**

### CODE REVIEW

File: src/Flickr.Net.Core.Test/BlubTest.cs
Comment (LineOfC0d3) [inline src/Flickr.Net.Core.Test/BlubTest.cs:1]: Ein sehr interessanter Test... muss ich schon sagen ;)

File: src/Flickr.Net.Core/Flickrs/Flickr_Upload_Async.cs [hunk 1]
```diff
@@ -156,45 +142,59 @@ private static async Task<string> UploadDataAsync(Stream imageStream, string fil
         //                bufferSize = bufferSize * 2;
```
Comment (LineOfC0d3) [inline src/Flickr.Net.Core/Flickrs/Flickr_Upload_Async.cs:142]: Can this commented out code be removed?

File: src/Flickr.Net.Core/Flickr.Net.Core.csproj [hunk 2]
```diff
@@ -25,6 +25,8 @@
         <PackageTags>Flickr;Api;Photo;Video</PackageTags>
         <PackAsTool>False</PackAsTool>
         <Company />
+        <GenerateDocumentationFile>True</GenerateDocumentationFile>
+        <DocumentationFile>G:\GIT\Flickr.Net\doc\SandCastleDocuments.xml</DocumentationFile>
```
Comment (LineOfC0d3) [inline src/Flickr.Net.Core/Flickr.Net.Core.csproj]: Is this a local path?

Comment (st0o0) [reply src/Flickr.Net.Core/Flickr.Net.Core.csproj]: yes can you make this relative?

File: src/Flickr.Net.Core/Flickrs/Flickr_Places_Async.cs [hunk 3]
```diff
@@ -0,0 +1,616 @@
...
+    /// Places ID.
+    /// </summary>
+    /// <param name="placeId">
+    /// A Flickr Places ID. (While optional, you must pass either a valid Places ID or a WOE ID.)
+    /// </param>
+    /// <param name="woeId">
+    /// A Where On Earth (WOE) ID. (While optional, you must pass either a valid Places ID or a WOE ID.)
+    /// </param>
+    /// <param name="cancellationToken"></param>
+    Task<PlaceCollection> GetChildrenWithPhotosPublicAsync(string placeId, string woeId = null, CancellationToken cancellationToken = default);
```
Comment (LineOfC0d3) [inline src/Flickr.Net.Core/Flickrs/Flickr_Places_Async.cs:438]: Wouldn't it be better to overload this funtion (and functions similar to this) to make it clear that either "_placeId_", "_woeId_" (or both if possible) have to be set.
Adding functions like
`Task<PlaceCollection> GetChildrenWithPhotosPublicAsync(string placeId, CancellationToken cancellationToken = default)`
and
`Task<PlaceCollection> GetChildrenWithPhotosPublicAsync(string woeId, CancellationToken cancellationToken = default)`
for clarity and internally using a 
`Task<PlaceCollection> GetChildrenWithPhotosPublicAsync(string placeId, string woeId, CancellationToken cancellationToken = default)`
 function to still have the functionality in one place. The last function can also be public if wanted.
 
 This also requires some kind of special PlaceId or WoeId object to to get different signatures...

**Pull request merged**